inst/extdata/gcam-usa/NREL_Com_PV_supply_curve.csv

```diff
@@ -0,0 +1,566 @@
+# File: NREL_Com_PV_supply_curve.csv
+# Title: PV supply curve for commercial buildings
+# Units: Rel_Cost - Unitless
+# Source: www.nrel.gov
```

Comment:
Please add a better source than just nrel.gov. The source is:

Denholm, P. & Margolis, R. Supply Curves for Rooftop Solar PV-Generated Electricity for the United States. National Renewable Energy Laboratory, Technical Report NREL / TP-6A0-44073 (2008)

If possible, please track the specific table in the report down.